clouddrift/ragged.py

```diff
@@ -637,10 +637,32 @@ def subset(
     >>> func = (lambda arr: ((arr - arr[0]) % 2) == 0)
     >>> subset(ds, {"time": func})
 
+    The filtering function can accept several input variables
+    (note that here, filtering is done along the trajectory dimension).
```

Comment:
What do you mean by "here"? In the example that follows because it pertains to the `start_lon` and `start_lat` variables which have dimensions `trajectory`? In that case that comment should probably be moved a few lines but I think it could be removed altogether.

Reply:
Perhaps line 634 above could start with: "You can also filter the data by applying a function acting on any of the variables."

Reply:
> What do you mean by "here"? In the example that follows because it pertains to the `start_lon` and `start_lat` variables which have dimensions `trajectory`? In that case that comment should probably be moved a few lines but I think it could be removed altogether.

Yes exactly.

By reading the docstring, I felt that it could be useful to highlight that the `Callable` criterion either apply along the `obs` dimension (through `apply_ragged`) or along the `trajectory` dimension by directly invoking the `Callable`.

But it is definetly up to you!